Client/Cards/index.ts

```diff
@@ -82,6 +82,12 @@ export class Cards extends List<
 		)
 		return result
 	}
+	async generateCard(providerCardId: string, providerCode: model.ProviderCode) {
+		const result = await this.connection.get<model.CardResponse>(
+			`cards/virtual/${providerCode}/${providerCardId}/generate`
+		)
+		return result
+	}
```

Comment:
should we be using v2? same for the others here

Reply:
yes we should, i will change/add that